src/src/index.tsx

```diff
@@ -3,14 +3,21 @@ import './css/main.css';
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 
-import {App} from './components';
-import {Contexts} from './components/Core/Contexts';
+import { App } from './components/App';
+import { Contexts } from './components/Core/Contexts';
 
-const root = ReactDOM.createRoot(document.getElementById('root')!);
-root.render(
-  <React.StrictMode>
-    <Contexts>
-      <App/>
-    </Contexts>
-  </React.StrictMode>
-);
+const container =
+  document.querySelector('header div[aria-label="Search"]')?.parentElement
```

Comment:
Could we maybe leave a comment describing what this `container` is ultimately supposed to be referring to?

Reply:
Good idea! Added in [fafba95](https://github.com/maxxxxxdlp/calendar-stats/pull/27/commits/fafba9599634c8637a73d17191c0f44691a3f30c)